frontend/src/pages/LogsPage/SearchForm/SearchForm.tsx

```diff
@@ -246,7 +246,7 @@ const Search: React.FC<{
 										<Stack direction="row" gap="8">
 											<Text>{activeTerm.value}:</Text>{' '}
 											<Text color="weak">
-												{activeTerm.key ?? 'Body'}
+												{activeTerm.key ?? 'Message'}
```

Comment:
Updates the combobox UI
<img width="609" alt="Screenshot 2023-03-07 at 3 32 25 PM" src="https://user-images.githubusercontent.com/58678/223569048-a4104ed3-af98-4a4d-ba81-4ac7d20311ef.png">